etc/wazo-auth/conf.d/50-wazo-default-policies.yml

```diff
@@ -26,13 +26,7 @@ default_policies:
       - 'chatd.users.me.rooms.#'
       - 'chatd.users.presences.read'
       - 'confd.infos.read'
-      - 'confd.users.me.#.read'
-      - 'confd.users.me.forwards.*.*'
-      - 'confd.users.me.funckeys.*'
-      - 'confd.users.me.funckeys.*.*'
-      - 'confd.users.me.read'
-      - 'confd.users.me.services.*.*'
-      - 'confd.users.me.update'
+      - 'confd.users.me.#'
```

Comment:
Warning: You give access to user to delete themself, which was not allowed before